Makefile

```diff
@@ -1,8 +1,9 @@
-.PHONY: package
+.PHONY: package build
```

Comment:
This should have always been there. It means you won't have to do `make build -B` to avoid the "`build` is up to date" error message; just `make build` will always work now.